ontology/owl/fdri-metadata.ttl

```diff
@@ -328,56 +417,64 @@ above the local reference ellipsoid).""" ;
                                      :EnvironmentalMonitoringNetwork
                                    )
-                     ] .
+                     ] ;
+          rdfs:label "Relates an Environmental Monitoring Programme to the Environmental Monitoring Facility or Environmental Monitoring Network used by the programme in the delivery of its outputs."@en ,
+                     "utilises"@en .
 
 
 ###  http://fdri.ceh.ac.uk/vocab/metadata/valueStatistic
 :valueStatistic rdf:type owl:ObjectProperty ;
-                rdfs:range :ValueStatistic .
+                rdfs:range :ValueStatistic ;
+                rdfs:comment "Relates an aggregated data resource to the type of aggregation applied."@en ;
+                rdfs:label "value statistic"@en .
 
 
 ###  http://fdri.ceh.ac.uk/vocab/metadata/variable
 :variable rdf:type owl:ObjectProperty ;
-          rdfs:range :Variable .
+          rdfs:range :Variable ;
+          rdfs:comment "Relates a Measure to the Variable that the is measured."@en ;
+          rdfs:label "measure"@en .
 
 
 ###  http://fdri.ceh.ac.uk/vocab/metadata/wasModifiedBy
 :wasModifiedBy rdf:type owl:ObjectProperty ;
-               rdfs:subPropertyOf <http://www.w3.org/ns/prov#wasInfluencedBy> ;
-               rdfs:domain :ObservationDataset .
+               rdfs:subPropertyOf prov:wasInfluencedBy ;
+               rdfs:domain :ObservationDataset ;
+               rdfs:comment "Relates a resource to the activity that modified the resource in some way."@en ;
+               rdfs:label "was modified by"@en .
 
 
 ###  http://purl.org/dc/terms/replaces
-<http://purl.org/dc/terms/replaces> rdf:type owl:ObjectProperty .
+dct:replaces rdf:type owl:ObjectProperty .
 
 
 ###  http://purl.org/dc/terms/type
-<http://purl.org/dc/terms/type> rdf:type owl:ObjectProperty .
+dct:type rdf:type owl:ObjectProperty .
 
 
 ###  http://schema.org/unit
-<http://schema.org/unit> rdf:type owl:ObjectProperty ;
-                         rdfs:domain <http://qudt.org/schema/qudt/Unit> .
+schema:unit rdf:type owl:ObjectProperty ;
+            rdfs:domain <http://qudt.org/schema/qudt/Unit> .
 
 
 ###  http://schema.org/valueReference
-<http://schema.org/valueReference> rdf:type owl:ObjectProperty .
+schema:valueReference rdf:type owl:ObjectProperty .
 
 
 ###  http://www.w3.org/2003/01/geo/wgs84_pos#location
-<http://www.w3.org/2003/01/geo/wgs84_pos#location> rdf:type owl:ObjectProperty .
+geo:location rdf:type owl:ObjectProperty .
 
 
 ###  http://www.w3.org/ns/prov#wasInfluencedBy
-<http://www.w3.org/ns/prov#wasInfluencedBy> rdfs:subPropertyOf <http://www.w3.org/ns/ssn/wasOriginatedBy> .
+prov:wasInfluencedBy rdfs:subPropertyOf ssn:wasOriginatedBy .
 
 
 ###  http://www.w3.org/ns/sosa/hasFeatureOfInterest
-<http://www.w3.org/ns/sosa/hasFeatureOfInterest> <http://schema.org/domainIncludes> :EnvironmentalMonitoringFacility .
+sosa:hasFeatureOfInterest schema:domainIncludes :EnvironmentalMonitoringFacility .
```

Comment:
Interesting, I hadn't twigged before that we were using `scheme:domainIncludes`

Reply:
I think I picked that up from looking at how it was done in the SOSA/SSN extensions